doc/nrf/working_with_nrf/nrf91/nrf9160.rst

```diff
@@ -286,6 +283,92 @@ For example, when building a non-secure application for nRF9160 DK v1.0.0, use `
 
 See :ref:`zephyr:application_board_version` and :ref:`zephyr:nrf9160dk_additional_hardware` for more information.
 
+.. _nrf9160_external_flash:
+
+External flash memory
+*********************
+
+The nRF9160 DK version 0.14.0 and later versions contain an 8 MB external flash (MX25R6435F).
+The external flash can be used to store data that does not fit on the internal flash.
+To free up GPIO when the external flash is not needed, the nRF9160 DK board controller (nRF52840 SoC) can control a switch to enable or disable routing between the external flash and the nRF9160 SIP.
+See `nRF9160 DK board controller <nRF9160 DK board control section in the nRF9160 DK User Guide_>`_ and `External memory <External memory section in the nRF9160 DK User Guide_>`_ sections in the nRF9160 DK user guides for more details.
+
+Programming the board controller firmware
+=========================================
+
+To use the external flash memory on the nRF9160 DK v0.14.0 or later versions, you must program the board controller firmware v2.0.1 or higher by completing the following steps:
+
+#. Download the nRF9160 DK board controller firmware from the `nRF9160 DK downloads`_ page.
+#. Make sure the **PROG/DEBUG SW10** switch on the nRF9160 DK is set to **nRF52**.
+#. Program the board controller firmware (:file:`nrf9160_dk_board_controller_fw_2.0.1.hex`) using the `Programmer app <Programming a Development Kit_>`_ in nRF Connect for Desktop.
+   By default, this enables pin routing to external flash.
+
+..
+   The new board controller firmware has been updated to enable routing by default (new DK versions?).
+..
+
+Adding the configuration and devicetree option
+===============================================
+
+The application also needs configuration and devicetree options to use the external flash.
```

Comment:
```suggestion
To use external flash, the application also needs to enable configuration and devicetree options.
```
Not quite happy with this version either, but it at least can't be read as devicetree options using external flash.